examples/secrets/BUILD.bazel

```diff
@@ -27,3 +34,13 @@ load("@com_adobe_rules_gitops//gitops:defs.bzl", "k8s_deploy")
         ("mynamespace", "dev-cluster"),
     ]
 ]
+
+# test expected transofmation results.
+write_source_files(
```

Comment:
where is the test that asserts the equality of these files and the rendered output

Reply:
write_source_fiels create the test and binary to update the golden file